tests/bdd/features/volume/topology/test_node-topology.py

```diff
@@ -82,7 +82,7 @@
         "node_name": NODE_1_NAME,
         "pool_uuid": NODE_1_POOL_1_UUID,
         "pool_body": CreatePoolBody(
-            ["malloc:///node1pool1?size_mb=50"],
+            ["malloc:///node1pool1?size_mb=32"],
```

Comment:
Since you are already here? Would it be a good idea to make it file?

Reply:
Probably yes, but would rather get this in first as I've already spent quite some time on it